web/problems/migrations/0011_auto_20160203_2250.py

```diff
@@ -14,18 +14,18 @@ class Migration(migrations.Migration):
             model_name="historicalproblem",
             name="language",
             field=models.CharField(
-                default=b"python",
```

Comment:
@mucamaca found out that CharField doesn't support being set to bytestring since the 1st of May 2017 and TextField something similar. Changed all relevant bytestrings to string in fba13de.